raiden/tests/integration/test_dispute.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
```

Comment:
Happened because of a rebase, cherry-picked the change and made an isolated PR https://github.com/raiden-network/raiden/pull/460 but forgot to drop the change, removing it.